src/ppx-metaquot/genlifter.ml

```diff
@@ -19,8 +19,21 @@ module Main : sig end = struct
 
   (*************************************************************************)
 
+  module Compat = struct
+#if OCAML_VERSION >= (4, 14, 0)
```

Comment:
just to be sure, these preprocessor clauses are based on cppo (?)

Reply:
Yup!